x-pack/filebeat/filebeat-input.yml

```diff
@@ -0,0 +1,5 @@
+- type: filestream
```

Comment:
What is this file and where is it used?

Reply:
sorry, dome debug stuff were left behind